app/views/catalog/_sort_widget.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if show_sort_and_per_page? and active_sort_fields.many? %>
```

Comment:
So this "tidying sort" for general results isn't in either issue, but it's an additional change kind of thing right? This is mainly to make the buttons for sort and "per page" full width on mobile? Was someone asking for this change?

Reply:
Right, this is kinda of an additional change. Since I changed the share widget to full width on mobile, I ended up changing per page button as well. Then since I changed the per page, it is also changed on search results as well so I thought I would change the sort to be full width so all are consistent.

Reply:
Yup, makes sense. And I know you showed it to me in a demo, I think anyway, but, I didn't realize until I saw the code.